src/e2e/scala/ie/zalando/fabric/gateway/spec/LegacyTlsValidationSpec.scala

```diff
@@ -8,6 +8,9 @@ import ie.zalando.fabric.gateway.{LoggingSttpBackend, TestConstants}
 import javax.net.ssl.{HttpsURLConnection, SSLContext, TrustManager, X509TrustManager}
 import org.scalatest._
 
+// Ignored because the client code that runs the tests can no longer create requests using the old TLS v1.1 version.
+// Annotation is still being added to the resources
```

Comment:
We can't have the integration tests depend on an old http client? Is this because of upgrading to java 11 or something? Could delete this test if so.

Reply:
I didn't touch of any code and the test stopped working. Not really sure why tbh. I was gonna leave the code there so that I could come back to it if I had time to fix later in the week...